contracts/exchangeIssuance/ExchangeIssuanceLeveraged.sol

```diff
@@ -217,6 +214,65 @@ contract ExchangeIssuanceLeveraged is ReentrancyGuard, FlashLoanReceiverBaseV2{
         _approveToken(_token);
     }
 
+    /**
+     * Gets the input cost of issuing a given amount of a set token. This
+     * function is not marked view, but should be static called from frontends.
+     * This constraint is due to the need to interact with the Uniswap V3 quoter
```

Comment:
Even if we were not interacting with UniV3 we still couldn't have this as view since we are syncing the aaveLeverage module. Up to you if  you want to adjust the comment.